src/sentry/snuba/subscriptions.py

```diff
@@ -161,6 +161,54 @@ def delete_snuba_subscription(subscription):
     :return:
     """
     subscription.update(status=QuerySubscription.Status.DELETING.value)
+
+    delete_subscription_from_snuba.apply_async(
+        kwargs={"query_subscription_id": subscription.id}, countdown=5
+    )
+
+
+def bulk_disable_snuba_subscriptions(subscriptions):
+    """
+    Disables a list of snuba query subscriptions.
+    :param subscriptions: The subscriptions to disable
+    :return:
+    """
+    for subscription in subscriptions:
+        # TODO: Batch this up properly once we care about multi-project rules.
+        disable_snuba_subscription(subscription)
+
+
+def disable_snuba_subscription(subscription):
+    """
+    Disables a subscription to a snuba query.
+    :param subscription: The subscription to disable
+    :return:
+    """
+    subscription.update(status=QuerySubscription.Status.DISABLED.value, subscription_id=None)
```

Comment:
If you set `subscription_id` to None here, then `delete_subscription_from_snuba` won't have an id to use to delete the subscription in snuba.

Reply:
I knew I should have written tests to actually verify the deletion of the snuba subscription... :(